libminifi/include/utils/ValueParser.h

```diff
@@ -119,6 +119,17 @@ class ValueParser {
     return *this;
   }
 
+  ValueParser& parse(double& out) {
+    double result;  // NOLINT
```

Comment:
Could we just initialize this to 0 to satisfy the linter?

Reply:
sure, fixed it in [007a486](https://github.com/apache/nifi-minifi-cpp/pull/1032/commits/007a486268761ce8fc086dbcc5148ff1e03b0689)